packages/deces-backend/tests/test_query_params.sh

```
@@ -66,7 +66,7 @@ else
     echo -e "\e[31mdeathDepartement: KO!\e[0m"
     exit 1
 fi
-if curl -s -XGET http://localhost:${BACKEND_PORT}/deces/api/v1/search?deathDate=2020\&firstName=Ana\&fuzzy=false | grep -q '"response":{"total":10' ; then
+if curl -s -XGET http://localhost:${BACKEND_PORT}/deces/api/v1/search?deathDate=2020\&firstName=Ana\&fuzzy=false | grep -Eq '"response":\{"total":[1-9][0-9]*' ; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)